test/onnx/test_fx_op_consistency.py

```diff
@@ -71,7 +71,10 @@
         "ceil", dtypes=onnx_test_common.BOOL_TYPES + onnx_test_common.INT_TYPES,
         reason=onnx_test_common.reason_onnx_does_not_support("Ceil")
     ),
-    xfail("unflatten", reason="AssertionError: Expected 1 inputs, got 3 (https://github.com/pytorch/pytorch/issues/99534)"),
+    skip(
```

Comment:
nit: shouldn't this (and above) be 'xfail'?

Reply:
Yea, you are right. I missed this one.

Reply:
Done